Dont use pixel formats that have been deprecated for years. This fixes the build
against libavutil 50.0.0


Index: xvidcap-1.1.7/src/xtoffmpeg.c
===================================================================
--- xvidcap-1.1.7.orig/src/xtoffmpeg.c
+++ xvidcap-1.1.7/src/xtoffmpeg.c
@@ -96,7 +96,7 @@
                         || (x)==PIX_FMT_YUVJ420P || (x)==PIX_FMT_YUVJ422P \
                         || (x)==PIX_FMT_YUVJ444P)
 
-#define PIX_FMT_ARGB32 PIX_FMT_RGBA32  /* this is just my personal
+#define PIX_FMT_ARGB32 PIX_FMT_RGB32   /* this is just my personal
                                         * convenience */
 
 /*
